### ISSUE TITLE
List branches sorted by last commit time

### ISSUE THREAD

**Issue by stanislavkozlovski** [bahmutov/git-branches#11]
It would be great if we could have the ability to get some ordering on our branches

Some projects have a lot of branches and it is hard to navigate through them. While it's obvious that it's bad practice to keep a lot of branches - you do not always have control over the repo.

**Comment by bahmutov** [comment]
Yup, completely agree. Probably I would want to order branches by last commit timestamp? 
```
old - oldest branch
....
recent - branch with the most recent commit
```

**Comment by stanislavkozlovski** [comment]
If the functionality is in place, I believe it doesn't hurt to have both possibilities (order ascending and descending).
Another proposal I have, probably worth a separate issue, is to have the ability to limit and maybe even set a config value for default number of branches shown. e.g show me the 10 branches it was last committed in
